src/as2fm/scxml_converter/scxml_entries/xml_utils.py

```diff
@@ -93,30 +97,45 @@ def read_value_from_xml_child(
     xml_child = xml_tree.findall(child_tag)
     if xml_child is None or len(xml_child) == 0:
         if not none_allowed:
-            print(f"Error: reading from {xml_tree.tag}: Cannot find child '{child_tag}'.")
+            log_error(
```

Comment:
This is neat! We should make a pass over the whole AS2FM and make sure we use it consistently (can also be done in another PR, in the future)! :)